python-modules/cis_aws/package-lock.json

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
I think we should probably not store this file, so that its recreated with up to date deps each time.
for ex the current one had an dependency loop with leveldown, but that would also ensure vulns to be patched automatically